src/main/java/com/griefcraft/lwc/LWC.java

```diff
@@ -921,19 +922,22 @@ public Player findPlayer(int minX, int maxX, int minY, int maxY, int minZ, int m
     }
 
     /**
-     * Send a locale to a player or console
+     * Check the data of locale
+     * Returns null if invalid
      *
-     * @param sender
-     * @param key
-     * @param args
+     * @param sender CommandSender
+     * @param key key of locale
+     * @param args Character to be rewritten
+     *              Example: %block% --> Chest
+     * @return message or null
      */
-    public void sendLocale(CommandSender sender, String key, Object... args) {
+    private String[] getLocaleMessage(CommandSender sender, String key, Object... args) {
```

Comment:
Are there any existing usages of this method? If not, changing it should be fine. However, a method to get a locale should *never* be issuing an LWCSendLocaleEvent (as a locale has not been sent yet). That should be done in the method where the actual message is sent, when it is determined that the message is actually being sent. This should be refactored immediately or it may cause issues in this plugin and others using the event.